diff -uNr v-0.1.11/compiler/main.v v-0.1.11.mod/compiler/main.v
--- v-0.1.11/compiler/main.v	2019-07-01 17:53:54.000000000 +0300
+++ v-0.1.11.mod/compiler/main.v	2019-07-03 15:30:02.009567393 +0300
@@ -492,7 +492,7 @@
 	} 
 	linux_host := os.user_os() == 'linux'
 	v.log('cc() isprod=$v.pref.is_prod outname=$v.out_name')
-	mut a := ['-w', '-march=native']// arguments for the C compiler
+	mut a := ['-w']// arguments for the C compiler
 	flags := v.table.flags.join(' ')
 	/* 
 	mut shared := ''
