diff -uNr haproxy-2.1.5/src/standard.c haproxy-2.1.5.mod/src/standard.c
--- haproxy-2.1.5/src/standard.c	2020-05-29 14:17:23.000000000 +0300
+++ haproxy-2.1.5.mod/src/standard.c	2020-06-02 01:36:25.654293514 +0300
@@ -4350,7 +4350,7 @@
 static int dladdr_and_size(const void *addr, Dl_info *dli, size_t *size)
 {
 	int ret;
-#ifdef __USE_GNU // most detailed one
+#ifdef defined(__USE_GNU) && !defined(__ANDROID__) // most detailed one
 	const ElfW(Sym) *sym;
 
 	ret = dladdr1(addr, dli, (void **)&sym, RTLD_DL_SYMENT);
