diff --git a/pkg/clipboard/clipboard.go b/pkg/clipboard/clipboard.go
index 937d5d3e..b6670dfe 100644
--- a/pkg/clipboard/clipboard.go
+++ b/pkg/clipboard/clipboard.go
@@ -14,7 +14,7 @@ import (
 
 var (
 	// ErrNotSupported is returned when the clipboard is not accessible
-	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install xsel or xclip or use -f to print to console")
+	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install termux-api or use -f to print to console")
 )
 
 // CopyTo copies the given data to the clipboard and enqueues automatic
