diff -uNr aptly-1.3.0/Makefile aptly-1.3.0.mod/Makefile
--- aptly-1.3.0/Makefile	2018-05-14 17:41:33.000000000 +0300
+++ aptly-1.3.0.mod/Makefile	2019-02-25 16:58:56.348102664 +0200
@@ -1,5 +1,5 @@
 GOVERSION=$(shell go version | awk '{print $$3;}')
-VERSION=$(shell git describe --tags | sed 's@^v@@' | sed 's@-@+@g')
+VERSION ?= $(shell git describe --tags | sed 's@^v@@' | sed 's@-@+@g')
 PACKAGES=context database deb files gpg http query swift s3 utils
 PYTHON?=python
 TESTS?=
